### PR TITLE
Fixed dynamic config reading

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
fix

### CODE REVIEW

**Review by lego963 — DISMISSED** [review]

**Review by outcatcher — APPROVED** [review]

**Comment by lego963** [comment]
recheck

**Review by outcatcher — DISMISSED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -30,6 +30,7 @@ pyyaml-typed==0.1.1
 requests==2.24.0
 requests-hawk==1.0.1
 rfc3339-validator==0.1.2
+setuptools==50.3.0
```
Comment (outcatcher) [inline requirements.txt:33]: Shouldn't it be in `dev-requirements.txt`?

**Review by lego963 — APPROVED** [review]

**Review by lego963 — APPROVED** [review]

**Comment by outcatcher** [comment]
recheck

**Comment by outcatcher** [comment]
recheck

**Comment by outcatcher** [comment]
recheck

**Review by lego963 — APPROVED** [review]

**Pull request merged**